model/registry/process.yaml

```diff
@@ -76,3 +76,38 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+  - id: registry.process.attributes
+    prefix: process
+    type: attribute_group
+    brief: >
+        Metric attributes that appear on some process metrics.
```

Comment:
Since this is in the registry, it's not about only metrics anymore. Either remove this or maybe change the sentence so it's "generic"

Reply:
I don't see these attributes being used on anything other than process metrics, so I'm not sure how to reword either one to be generic. These are attributes that appear on some process metrics, and that's probably it. How should this be reworded?

Reply:
Hum.. the whole point of the registry is for re-usability. Then I'm wondering if we really need to move all of these attributes to the registry? 🤔 I'm actually now wondering where the need for this came now. I don't think other metrics have their attributes defined in the registry. Could you clarify maybe?

Reply:
This was requested here: https://github.com/open-telemetry/semantic-conventions/pull/330#discussion_r1419228700

Reply:
> Hum.. the whole point of the registry is for re-usability. Then I'm wondering if we really need to move all of these attributes to the registry? 🤔 I'm actually now wondering where the need for this came now. I don't think other metrics have their attributes defined in the registry. Could you clarify maybe?

We are currently requiring all attributes in semconv to be part of the registry.  This is to enable x-signal journeys where something my start as an EVENT then become a METRIC.   

I think your concern is related to https://github.com/open-telemetry/semantic-conventions/issues/394.  Let's fix that issue, but no block metrics moving their attributes to the registry.

Reply:
I submitted https://github.com/open-telemetry/semantic-conventions/pull/786 for it. But it is still unclear tbh. If metric attributes are "general", like `cpu.state`, which is used in `process` and `system` metrics. Should this go under a `cpu` namespace? Because here it is under `process.cpu.state` which cannot be used in system metrics.

Reply:
`system.cpu.state` and `process.cpu.state` are intentionally different because despite sharing a name, their enums are distinct from one another.

Reply:
Would by any chance make sense to unify them? Related to https://github.com/open-telemetry/semantic-conventions/pull/765#discussion_r1506373063

Reply:
Having one's allowed values being a subset of the other one's values makes sense to me. Not sure if this can somehow be "templated" with SemConv's rulings.

Reply:
@braydonk I have filed https://github.com/open-telemetry/semantic-conventions/issues/840 for this one.